### PR TITLE
Reduce usage of annotations 'choice' and 'choicesAllMatching'

### PR DESCRIPTION
**Pull request by tobolar**
Use 'choice' and 'choicesAllMatching' annotations only where reasonable.

### CODE REVIEW

File: ThermofluidStream/Interfaces/SISOFlow.mo [hunk 1]
```diff
@@ -30,12 +30,12 @@ partial model SISOFlow "Base Model with basic flow eqautions for SISO"
   Outlet outlet(redeclare package Medium=Medium)
     annotation (Placement(transformation(extent={{80,-20},{120,20}})));
 
-SI.MassFlowRate m_flow(stateSelect=m_flowStateSelect) = inlet.m_flow "mass flow through component";
-
-// changing pressure calculation
-SI.Pressure dr_corr; // delta = out - in
-SI.Pressure dp; // delta = out - in
+  SI.MassFlowRate m_flow(stateSelect=m_flowStateSelect) = inlet.m_flow
+    "Mass flow through component";
```
Comment (nieweber) [inline ThermofluidStream/Interfaces/SISOFlow.mo:34]: Intended line break?

File: ThermofluidStream/Sensors/Internal/Types.mo [hunk 2]
```diff
@@ -65,8 +65,10 @@ package Types "Types used in the Sensor Package"
   end MassFlowUnit;
 
   type InitializationModelSensor = enumeration(
-      steadyState "Steady state initialization",
-      state "Initial output state") "Initaization modes for sensor lowpass";
+    steadyState
+      "Steady state initialization (derivatives of states are zero)",
+    state
+      "Initialization with initial output state") "Initaization modes for sensor lowpass";
```
Comment (nieweber) [inline ThermofluidStream/Sensors/Internal/Types.mo:71]:
```suggestion
      "Initialization with initial output state") "Initialization modes for sensor lowpass";
```

File: ThermofluidStream/Utilities/Types.mo [hunk 3]
```diff
@@ -2,25 +2,23 @@ within ThermofluidStream.Utilities;
...
-      none)
+    none
+      "No initialization",
+    steadyState
+      "Steady state initialization (derivatives of states are zero)",
+    state
+      "Initialization with initial states",
+    derivative
+      "Initialization with initial derivatives of states")
     "Choices for Initialization of a state."
```
Comment (nieweber) [inline ThermofluidStream/Utilities/Types.mo:13]:
```suggestion
    "Choices for initialization of a state."
```

File: ThermofluidStream/Interfaces/SISOFlow.mo (hunk 1 above)
Comment (tobolar) [reply ThermofluidStream/Interfaces/SISOFlow.mo:34]: More or less. But I agree there is no rule for maximum line length. So we can revert this line break.

Comment (nieweber) [reply ThermofluidStream/Interfaces/SISOFlow.mo:34]: Okay, I was just wondering if Dymola once again caused some unwanted line breaks :)

**Review by nieweber — APPROVED** [review]

File: ThermofluidStream/Interfaces/SISOFlow.mo (hunk 1 above)
Comment (tobolar) [reply ThermofluidStream/Interfaces/SISOFlow.mo:34]: No no, it was really me. :-|

**Pull request merged**